specification/batch/Azure.Batch/routes.tsp

```diff
@@ -0,0 +1,2215 @@
+>;
+
+#suppress "@azure-tools/typespec-azure-core/no-response-body" "Legacy behavior"
+@head op ResourceInfoOperation<TResource, TResponse> is RpcOperation<
+  TResource & BatchClientRequestHeaders,
+  BatchResponseHeaders & TResponse,
+  NoFilter,
+  BatchError
+>;
+
+#suppress "@azure-tools/typespec-azure-core/no-error-status-codes" "Azure REST API guidelines recommend using 'default' error response for all error cases. Avoid defining custom 4xx or 5xx error cases."
+#suppress "@azure-tools/typespec-azure-core/no-response-body" "Legacy behavior"
+@doc("New rpc operation allowing multiple repsonses")
+@Foundations.Private.needsRoute
+@head
+op ResourceInfoOperationWithAdditionalResponse<
+  TParams extends TypeSpec.Reflection.Model,
+  TResponse extends TypeSpec.Reflection.Model,
+  TAdditionalResponse extends {} = {}
+> is Foundations.Operation<
+  TParams,
+  TResponse | TAdditionalResponse,
+  NoFilter,
+  BatchError
+>;
+
+#suppress "@azure-tools/typespec-azure-core/no-response-body" "Legacy behavior"
+@post op PostOperation<TResource, TResponse> is RpcOperation<
+  TResource & BatchClientRequestHeaders,
+  BatchResponseHeaders & TResponse,
+  NoFilter,
+  BatchError
+>;
+
+alias NoFilter = {};
+alias MinimalMetadata = {
+  @doc("Type of content")
+  @header("content-type")
+  contentType: "application/json; odata=minimalmetadata";
+};
+
+alias Created<T> = {
+  @doc("A process exit code.")
+  @statusCode
+  code: "201";
+
+  @header("DataServiceId")
+  @doc("The OData ID of the resource to which the request applied.")
+  dataServiceId: string;
+
+  ...T;
+};
+
+// Interfaces ////////////////////
+
+#suppress "@azure-tools/typespec-azure-core/no-response-body" "The body of non-204 responses should not be empty."
+#suppress "@azure-tools/typespec-azure-core/no-error-status-codes" "Azure REST API guidelines recommend using 'default' error response for all error cases. Avoid defining custom 4xx or 5xx error cases."
+#suppress "@azure-tools/typespec-azure-core/no-rpc-path-params" "Operations defined using BatchOperation should not have path parameters."
+#suppress "@azure-tools/typespec-azure-core/casing-style" "The names of Model types must use camelCase"
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Operation 'Exists' should be defined using a signature from the Azure.Core namespace."
+#suppress "@azure-tools/typespec-azure-core/rpc-operation-request-body" "BatchOperation with '@get' cannot have a body."
+interface Batch {
+  @summary("Lists all of the applications available in the specified Account.")
+  @doc("""
+This operation returns only Applications and versions that are available for
+use on Compute Nodes; that is, that can be used in an Package reference. For
+administrator information about applications and versions that are not yet
+available to Compute Nodes, use the Azure portal or the Azure Resource Manager
+API.
+""")
+  @route("/applications")
+  @projectedName("java", "listApplicationsInternal")
```

Comment:
Not sure why every op renamed to internal in Java.

Reply:
The issue is that we want to use these names as the public names, but the generated signature isn't what we want (options bags vs positional arguments).

Reply:
Actually, scratch that. This should be fine **because** they have different signatures. Great suggestion - I'll make this change since @wiboris is currently on vacation.

Reply:
All that said, I did add @access() to all our operations since we definitely don't want to expose the internal methods.

Reply:
You already enabled the `partial-update` feature in config, so one way just remove the `@Generated` annotation over e.g. 
`pyublic PagedIterable<BatchJob> listJobs(Integer timeOutInSeconds, OffsetDateTime ocpdate, Integer maxresults, String filter, List<String> select, List<String> expand)`
and modify the method directly. Codegen would keep it from been overwritten.
ref https://github.com/Azure/azure-sdk-for-java/wiki/TypeSpec-Java-Quickstart#partial-update

`@access` etc. had to be done in client.tsp

Also you may want to have `@access(Access.public, "java")` to the model that you need to move back to public (`Access.internal` would make the model internal as well, if they are only used by this internal operation).

Reply:
added @access(Access.public, "java") to the models